examples/any-type/any_type.bal

```diff
@@ -1,21 +1,40 @@
 import ballerina/io;
 
+type Person object {
+    string fname;
+    string lname;
+
+    function __init(string fname, string lname) {
+        self.fname = fname;
+        self.lname = lname;
+    }
+
+    function getFullName() returns string {
+        return self.fname + " " + self.lname;
+    }
+};
+
 // This function returns a value of the `any` type.
 function getValue() returns any {
     string name = "cat";
     return name;
 }
 
 public function main() {
-    // In this example, the variable named `a` of the `any` type holds an `int` value.
-    any a = 5;
-    io:println(a);
+    // In this example, the variable named `a` of the `any` type holds
+    // a `Person` object.
+    any a = new Person("John", "Doe");
+
+    // Before anything useful can be done with `a`, we need to ascertain
+    // its type. To this end, a type assertion or a type guard can be used.
+    Person john = <Person>a;
+    io:println("Full name: ", john.getFullName());
 
-    // First, the variable of type `any` needs to be asserted to be of the required type.
-    int intVal = <int>a;
-    io:println(intVal + 10);
+    if (a is Person) {
+        io:println("First name: ", john.fname);
+    }
 
-    // In Ballerina, a variable of the `any` type can hold values of any type.
+    // Variables of `any` type can hold arrays as well.
```

Comment:
Could this be confusing? Specifically mentioning arrays? What we want to say is it can hold anything other than errors right?

Reply:
Yes, I guess arrays were taken as an example of a more complex of a type which can be held by an `any` variable

Reply:
```suggestion
    // Variables of type `any` can hold values of any type except for `error`.
```